code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -665,34 +665,25 @@
 	id = "borg_upgrade_selfrepair"
 	build_type = MECHFAB
 	build_path = /obj/item/borg/upgrade/selfrepair
-	materials = list(MAT_METAL=15000, MAT_GLASS=15000)
-	construction_time = 120
+	materials = list(MAT_METAL = 15000, MAT_GLASS = 15000)
+	construction_time = 80
 	category = list("Cyborg Upgrade Modules")
 
 /datum/design/borg_upgrade_expandedsynthesiser
 	name = "Cyborg Upgrade (Hypospray Expanded Synthesiser)"
 	id = "borg_upgrade_expandedsynthesiser"
 	build_type = MECHFAB
 	build_path = /obj/item/borg/upgrade/hypospray/expanded
-	materials = list(MAT_METAL=15000, MAT_GLASS=15000, MAT_PLASMA=5000)
-	construction_time = 120
-	category = list("Cyborg Upgrade Modules")
-
-/datum/design/borg_upgrade_highstrengthsynthesiser
```

Comment:
why did you remove this

Reply:
He said he was merging it with the other synthesiser upgrade in the description.

Reply:
There are a few other changes that has no mention. This one does but the ones that are getting changed are mainly cheaper.